website/events/templates/events/event.html

```diff
@@ -155,15 +155,25 @@ <h1 class="text-center section-title">{{ event.title }}</h1>
                                     </form>
                                 {% endif %}
                                 {% if permissions.create_registration %}
-                                    <p>
-                                        {% url 'singlepages:event-registration-terms' as terms_url %}
-                                        {% blocktrans trimmed %}
-                                            By registering,
-                                            you confirm that you have read the
-                                            <a target="_blank" href="{{ terms_url }}">terms and conditions</a>,
-                                            that you understand them and that you agree to be bound by them.
-                                        {% endblocktrans %}
-                                    </p>
+                                    {% if event.registration_required %}
+                                        <p>
+                                            {% url 'singlepages:event-registration-terms' as terms_url %}
+                                            {% blocktrans trimmed %}
+                                                By registering,
+                                                you confirm that you have read the
+                                                <a target="_blank" href="{{ terms_url }}">terms and conditions</a>,
+                                                that you understand them and that you agree to be bound by them.
+                                            {% endblocktrans %}
+                                        </p>
+                                    {% else %}
+                                        <p>
+                                            {% blocktrans trimmed %}
+                                                Even though registration is not required for this event,
+                                                you can still register to give an indication of who will be there
+                                                as well as mark the event as 'registered' in your calendar.
```

Comment:
This is a really nice solution to solve the problem I call 'is this even a registration' 👌